projects/js-packages/components/components/qr-post/index.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
Could we get rid of the react imports if we use this in the editor context with WordPress elements?